makefiles/build.mk

```diff
@@ -49,17 +47,30 @@ PROGRAM_TGT := $(PROGRAM).$(CURRENT_TARGET)
 
 SOURCES := $(wildcard $(SRCDIR)/*.c)
 SOURCES += $(wildcard $(SRCDIR)/*.s)
+SOURCES += $(wildcard $(SRCDIR)/*.asm)
 
 # allow for a src/common/ dir and recursive subdirs
+ifeq ($(CURRENT_TARGET),pmd85)
+SOURCES += $(call rwildcard,$(SRCDIR)/common/,*.asm)
```

Comment:
I'd prefer this was using a variable that is defined for pmd85 as ".asm"
See https://github.com/FujiNetWIFI/fujinet-lib/blob/main/makefiles/build.mk#L11 which Eric made into ASMEXT and defined for the compiler